java/integTest/src/test/java/glide/standalone/CommandTests.java

```diff
@@ -394,4 +394,56 @@ public void functionLoad() {
         assertEquals(libName, regularClient.functionLoad(newCode, true).get());
         // TODO test with FCALL
     }
+
+    @Test
+    @SneakyThrows
+    public void copy() {
```

Comment:
we should only need to run copy_with_dbdestination tests... the rest is already run in SharedCommentTests, right?